doc/input-output-reference/src/overview/group-location-climate-weather-file-access.tex

```diff
@@ -839,7 +839,7 @@ \subsubsection{Inputs}\label{inputs-7-013}
 
 \subsection{WeatherProperty:SkyTemperature}\label{weatherpropertyskytemperature}
 
-Sky Temperature, or radiative sky temperature, is internally calculated by EnergyPlus using an algorithm using horizontal infrared radiation from sky, cloudiness factors and current temperature. The algorithm is fully described in the Engineering Reference document. For flexibility, the following object can be entered to override the internal calculations. Much of the literature describes the sky temperature as relative to either drybulb or dewpoint temperature.
+Sky Temperature, or radiative sky temperature, is internally calculated by EnergyPlus using an algorithm using (1) horizontal infrared radiation or (2) an empirical model using sky cloudiness factors and current clear sky emissivity. By default, EnergyPlus calculates clear sky emissivity using Clark-Allen model, and its algorithm is fully described in the Engineering Reference document. For flexibility, the following object can be entered to adopt alternative sky emissivity calculation methods, or to override the entire internal sky temperature calculation from schedule import. Alternative methods of sky emissivity calculation include the calibrated forms of Martin \& Berdahl, Brunt, and Idso model, and their algorithms are described in the Engineering Reference as well. Much of the literature describes the sky temperature as relative to water vapor pressure, drybulb or dewpoint temperature.
```

Comment:
This sentence reads very differently from the original regarding horizontal infrared radiation. The original implies HorizIR is used in conjunction with the empirical model, but the new text implies HorizIR is a separate method (which doesn't have an option) so I'm confused by the revision.

Oh, I see from the code that it will use horizontal IR if present in the weather data, if one of the models is selected, but not if it's scheduled. That needs to be much more clear here. And should there be an option to always use the model even when horizontal IR is present?  Not sure which way I lean on that question.

Reply:
The logic is re-illustrated.